### PR TITLE
Update EditionNavBar.html

### LINKED ISSUE #6814: Broken edition pages in different languages
The edition pages are broken in languages other than English displaying the following message:

"/openlibrary/openlibrary/templates/type/edition/view.html: error in processing template: TypeError: Plural value must be an integer, got str (falling back to default template)
Unable to render this page. "

### Evidence / Screenshot (if possible)
screenshot

### Relevant url?

-- > https://openlibrary.org/books/OL27902091M/Carrie?lang=en (OK!)
-- > https://openlibrary.org/books/OL27902091M/Carrie?lang=es (You get an error)

### Steps to Reproduce

1. Go to any edition page
2. Try a language other than English. For example, by adding "?lang=es" to any edition page URL or by selecting a different language from the menu. You will get an error.

* Actual: The edition pages are not displayed in languages other than English.
* Expected: The editions pages should be displayed in any default or user-selected language.

### Details

- **Logged in (Y/N)?** Y
- **Browser type/version?** Firefox 103.0
- **Operating system?** Ubuntu
- **Environment (prod/dev/local)?** prod

### PR DESCRIPTION
**Pull request by mekarpeles**
Closes #6814

### CODE REVIEW

File: openlibrary/macros/EditionNavBar.html [hunk 1]
```diff
@@ -16,7 +16,10 @@
   </li>
   <li>
     <a href="#reader-observations" data-ol-link-track="BookPageNav|Reviews">
-      $ungettext('%(reviews)s Review', '%(reviews)s Reviews', total_reviews, reviews=total_reviews)
+      $if total_reviews > 1:
```
Comment (mekarpeles) [inline openlibrary/macros/EditionNavBar.html:19]:
```suggestion
      $if total_reviews > 0:
```

**Comment by mekarpeles** [comment]
Please squash when merging

**Comment by mekarpeles** [comment]
@jimchamp notes that even after these fixes, the plural i18n may not be available in all languages, e.g.
image
image

**Review by jimchamp — APPROVED** [review]
Lgtm!  Patch deployed.

**Pull request merged**